server/lib/configuration.js

```diff
@@ -494,6 +501,12 @@ var conf = module.exports = convict({
     },
     sample_rate: 1
   },
+  sync_tokenserver_url: {
+    default: 'http://127.0.0.1:5000/token',
+    doc: 'The url of the Firefox Sync tokenserver',
+    env: 'SYNC_TOKENSERVER_URL',
```

Comment:
need to add this to fxa-dev and the deploy doc.

im wondering if there can be a test that can verify that this is configured properly, because we are not using this for anything on the server and will only know that it broke via FF desktop